### ISSUE TITLE
Some questions regarding experiment setups

### ISSUE THREAD

**Issue by Henrymachiyu** [ruixin31/Spurious_Rewards#7]
Hi, 

Thank you for the amazing work! I am trying to reproduce the results based on the code. And I am currently running rlvr_deepscaler_grpo_llama_base_ground_truth.sh script with similar amount of gpus. Without changing anything, I noticed that it requires a total of 619 steps per episode, and 200 episodes in total. However, I noticed that in the paper, none of the plots are beyond 300 training steps. I wonder if there is anything wrong with the current setup?

Episode [1/200]:   2%|▏         | 12/629 [47:35<40:02:15, 233.61s/it, policy_loss=-0.0774, actor_lr=5e-7, kl=0, reward=0.338, response_length=1.03e+3, total_length=1.13e+3, return=6.48e-5]

Thank you in advance!

**Comment by RulinShao** [comment]
Hi @Henrymachiyu !

Thanks for your interest! You are all correct! We used a constant learning rate for our experiments, so we kept the large number of episodes to make the runs non-stopping unless we terminated them. We kept some runs going beyond 1k steps and found the performance stayed constant after early convergence in our setup, so we reported up to 300 training steps in the paper to reduce the computational cost for massive runs. Please let us know if you have any questions about it!
